### DISCUSSION TITLE
I've got RuntimeError when try to use controlnet

### DISCUSSION THREAD

**Discussion by IgorDyx** [Mikubill/sd-webui-controlnet#322] [Q&A]
RuntimeError: mat1 and mat2 shapes cannot be multiplied (77x1024 and 768x320)
I did everything in guide. What did I wrong? Can you help me? numbers in brackets are different every time.

**Comment by Mikubill** [comment]
check config file path in settings

**Reply by IgorDyx** [reply]
Everything is ok there. This error occurs only with SD 2.1, with 1.5 works nice.
